Formula/orientdb.rb

```diff
@@ -39,4 +39,14 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    begin
+      system "#{bin}/orientdb", "start"
```

Comment:
This line should be outside of this block, immediately preceding the `begin`. The reason for that is that if the start fails already, there's no point in running the `ensure` and attempting to stop the server that has never been started. (Another subtlety revealed by the currently failing test.)